images/eslint/sider_recommended_eslint.yml

```
@@ -0,0 +1,112 @@
+root: true
+
+parserOptions:
+  ecmaVersion: 2015
+
+extends:
+  - eslint:recommended
+
+rules:
+  # Sider's recommended ruleset is based on eslint:recommended,
+  # and following rules are additionally enabled/disabled.
+
+  # Disabled Variables
+  no-undef: off
+  no-unused-vars: off
+
+  # Enabled Possible Errors
+  no-await-in-loop: [error]
```

[question] What reason for using `[error]` instead of `error`?
`error` seems better, considering consistency with `off`.

```suggestion
  no-await-in-loop: error
```

---

OK, I fixed it.